### PR TITLE
feat: As a user, I want to see the navigation bar and wallet connection sections on Explorer

### PR DESCRIPTION
**Pull request by Solniechniy**
### Related ticket
https://github.com/Consensys/linea-attestation-registry/issues/441
### Type of change

- [x] Chore
- [ ] Bug fix
- [x] New feature
- [ ] Documentation update

## Check list

- [ ] Unit tests for any smart contract change
- [ ] Contracts and functions are documented

### CODE REVIEW

File: explorer/src/pages/Attestation/AttestationInfo/index.tsx [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
-import { createDateListItem } from "./utils";
-import { Hex, hexToNumber } from "viem";
 import { toModuleById } from "@/routes/constants";
 import { cropString } from "@/utils/stringUtils";
-import { DASH } from "@/constants";
+import { Link } from "@/components/Link";
+import { createDateListItem } from "./utils";
 
 export const AttestationInfo: React.FC<Attestation> = ({ ...attestation }) => {
   const { attestedDate, expirationDate, revocationDate, id, revoked, attester, portal, subject } = attestation;
```
Comment (Solniechniy) [inline explorer/src/pages/Attestation/AttestationInfo/index.tsx:12]: @alainncls @orbmis 
We've find out that attestedDate, expirationDate, revocationDate came as string, not as number type what declared in OnChainAttestation, could you look into? 
Version sdk 0.0.15 
where you will better to write about some questions like that?

Comment (alainncls) [reply explorer/src/pages/Attestation/AttestationInfo/index.tsx:12]: Oops! Yes, this is an issue!  
In such cases you can open an issue on the [Project Board](https://github.com/orgs/Consensys/projects/17/views/2) and/or ping us on [Discord](https://discord.gg/Sq4EmYdBEk)!  
Anyway, I'll fix that in a future release of the SDK, promise!

File: explorer/src/hooks/useWindowDimensions.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  }
+
+  return dimensions;
+}
+
+const useWindowDimensions = () => {
+  const [windowDimension, setWindowDimension] = useState<typeof initialDimensions>(getWindowDimensions());
+
+  useEffect(() => {
+    const handleResize = () => {
```
[the remark this answers is not in the document]
Comment (Solniechniy) [reply explorer/src/hooks/useWindowDimensions.ts:45]: Seems that useWindowDimensions created there once and then exported, so each useWindowDimensions() call will return the same reference. UseEffect don't have any dependency array so it should fire only once while initialisation, so it shouldn't call handleResize more than once. Your comment on right direction, but seems that only in that way it's not change any productivity. I can change it or leave, as u wish.

**Review by ars9 — APPROVED** [review]
All right, let's go!

**Pull request merged**